sympy/parsing/tests/test_sympy_parser.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
Does pytest complain about the encoding as well?

Reply:
It's not pytest complaining. It's Python 2.7:

```console
$ python
Python 2.7.10 (default, Oct  6 2017, 22:29:07) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sympy.parsing.tests.test_sympy_parser
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "sympy/parsing/tests/test_sympy_parser.py", line 218
SyntaxError: Non-ASCII character '\xce' in file sympy/parsing/tests/test_sympy_parser.py on line 218, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

I'm not sure why this error doesn't show up when using `bin/test`